packages/components/src/base-control/stories/index.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from '..';
+import Button from '../../button';
+
+const meta: ComponentMeta< typeof BaseControl > = {
+	title: 'Components/BaseControl',
+	component: BaseControl,
+	argTypes: {
+		help: { control: { type: 'text' } },
+		label: { control: { type: 'text' } },
```

Comment:
Forgot to mention, we should probably remove the ability to set a custom value for the `children` props through controls

Reply:
Good catch. I opted to address this by removing the not-so-necessary description for the `children` prop https://github.com/WordPress/gutenberg/pull/39468/commits/6e59dbe480f162683504cdb3c4fa691e6e43d0d6 (the `children` prop will automatically be hidden from the args table if there is no description). Let me know if you prefer to keep the description and hide the control explicitly in Storybook.

I also noticed that `children` should be a required prop, so I fixed that too https://github.com/WordPress/gutenberg/pull/39468/commits/1d505cb230fb8b3f06601c7422af66d63af61598.

Reply:
> Let me know if you prefer to keep the description and hide the control explicitly in Storybook.

This is a decision that we should make and apply package-wise. Personally, since certain components do not accept `children`, I think that we should list the `children` prop when it's available — but of course I'm open to hear other opinions!

Reply:
You're right. That would indeed be better for consistency. I'll put the description back, and just disable the control in this case.